projects/addon-editor/components/editor/editor.component.ts

```diff
@@ -161,6 +161,14 @@ export class TuiEditorComponent
         this.editor?.unsetLink();
     }
 
+    focus(): void {
```

Comment:
we have `onActiveZone` method above, doesn't it work if just add 
`if (focused) {this.editor?.focus())` in this method?

Reply:
got it